### DISCUSSION TITLE
Adding additional Beets library for FLAC after starting out with mp3?

### DISCUSSION THREAD

**Discussion by gbull1993** [beetbox/beets#4417] [Q&A]
Hi guys. Absolute newbie here. Running as docker container on Unraid. As the title says, I have made an existing library of mp3's and had set music to auto convert to mp3 via the convert plugin. However, I would like to make this my secondary Beets library and I would like to make a primary library consisting of FLAC media. I have checked the docs for this but am still quite unsure on how to achieve this. Any help would be greatly appreciated.

**Comment by sampsyo** [comment]
I believe this may be part of the goals of [beets-alternatives](https://github.com/geigerzaehler/beets-alternatives)?
